api/server.go

```diff
@@ -221,6 +221,7 @@ func RunServer(dry bool) http.Handler {
 	m.Add("1.0", "Post", "/teams", AuthorizationRequiredHandler(createTeam))
 	m.Add("1.0", "Delete", "/teams/{name}", AuthorizationRequiredHandler(removeTeam))
 	m.Add("1.4", "Post", "/teams/{name}", AuthorizationRequiredHandler(updateTeam))
+	m.Add("1.0", "Get", "/teams/{name}", AuthorizationRequiredHandler(teamInfo))
```

Comment:
this should be "1.4" instead of "1.0" as the route is being added on tsuru v1.4.